### ISSUE TITLE
release 1.6.0-Jakarta to Maven central

### ISSUE THREAD

**Issue by weberjn** [StripesFramework/stripes#112]
Could you please release 1.6.0-Jakarta to Maven central? 

This would help [Apache JSPWiki](https://github.com/apache/jspwiki)'s migration to Jakarta.

**Comment by philmadden83** [comment]
or @rgrashel release https://github.com/StripesFramework/stripes/pull/100 please

**Comment by jvaldezwm** [comment]
Found this:  https://mvnrepository.com/artifact/com.manydesigns/stripes

**Comment by rgrashel** [comment]
I'll see if I can somehow get access. I tried about six months ago and was
not able to get it to authenticate me.

-- Rick

**Comment by jvaldezwm** [comment]
> I'll see if I can somehow get access. I tried about six months ago and was
> not able to get it to authenticate me.
> […](#)

Thank you!

**Comment by kelunik** [comment]
We’ve now published a new version in Maven Central that’s using Jakarta: https://central.sonatype.com/namespace/org.stripesframework

**Comment by romank8k** [comment]
@kelunik is that a fork of this project? I have some projects to migrate and not clear which will be maintained going forward.
Also see https://github.com/StripesFramework/stripes/discussions/107

**Comment by kelunik** [comment]
@romank8k Yes, it's a fork. We haven't been able to reach any of the original maintainers at that time. I've also reached out to @rgrashel now that I've seen his Jakarta release.

**Comment by romank8k** [comment]
@kelunik anything you can tell us about the status of that fork?
I see there's been some work there recently but hard to follow as there are no release notes.
Is development being sponsored by Chrono24? (as I see it's under their org)
